scalyr_agent/scalyr_monitor.py

```diff
@@ -200,6 +200,17 @@ def increment_counter(self, reported_lines=0, errors=0):
         self.__errors += errors
         self.__lock.release()
 
+    def monitor_manager( self, manager ):
```

Comment:
Let's call this `config_from_monitors`.  At least it will better describe what this is used for.

Reply:
Done